test/SILGen/magic_identifier_file.swift

```diff
@@ -1,8 +1,28 @@
-// RUN: %target-swift-emit-silgen -module-name Foo %/s | %FileCheck --check-prefixes=BOTH,ABSOLUTE %s
-// RUN: %target-swift-emit-silgen -enable-experimental-concise-pound-file -DNEEDS_CONCISE -module-name Foo %/s | %FileCheck --check-prefixes=BOTH,CONCISE %s
+// RUN: %empty-directory(%t)
 
-// FIXME: Once this feature becomes non-experimental, we should update existing
-// tests and delete this file.
+// Build library in Swift 5 mode:
+// RUN: %target-build-swift-dylib(%t/%target-library-name(MagicIdentifierFileSwift5)) -module-name MagicIdentifierFileSwift5 -emit-module-path %t/MagicIdentifierFileSwift5.swiftmodule -emit-module-interface-path %t/MagicIdentifierFileSwift5.swiftinterface -swift-version 5 -enable-library-evolution %S/Inputs/MagicIdentifierFileSwift.swift
+
+// Build library in "Swift 6" mode:
+// RUN: %target-build-swift-dylib(%t/%target-library-name(MagicIdentifierFileSwift6)) -module-name MagicIdentifierFileSwift6 -emit-module-path %t/MagicIdentifierFileSwift6.swiftmodule -emit-module-interface-path %t/MagicIdentifierFileSwift6.swiftinterface -swift-version 5 -enable-experimental-concise-pound-file -enable-library-evolution %S/Inputs/MagicIdentifierFileSwift.swift
```

Comment:
Could you add TODO comment for replacing `-swift-version 5 -enable-experimental-concise-pound-file` with `-swift-version 6`?

Reply:
Sure. I've filed https://bugs.swift.org/browse/SR-13199 to use in TODO comments like this one.

Reply:
Done.